src/sc_crawler/workload_profiles.py

```
@@ -0,0 +1,364 @@
+                config_filter={"cores": "Multi-Core Performance"},
+            ),
+        ],
+    ),
+    "cicd": Workload(
+        name="CI/CD build",
+        rationale=dedent(
+            """
+            Build performance is driven by single-thread speed (many build tools are serial or poorly parallelised),
+            multi-core compilation throughput, compression and archiving for packaging, and text/scripting processing.
+            Geekbench Clang directly compiles the Lua interpreter, providing a strong proxy for real CI workloads.
+            Research shows modern CI runners are 1.5–2× faster with higher single-thread clocks, motivating the chosen weights.
```

Ruff (0.15.9)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)